src/Tests/ValidUntilTest.cs

```diff
@@ -1061,7 +1061,7 @@ public string GetSharedAccessSignature(SharedAccessBlobPolicy policy, SharedAcce
         public StorageUri SnapshotQualifiedStorageUri { get; }
         public CopyState CopyState { get; }
         public BlobType BlobType { get; }
-#elif NETCOREAPP2_0
+#elif NETCOREAPP2_1
```

Comment:
Can `NETCOREAPP2_1` be replaced with `NETCOREAPP`? Would remove the need to updated it every time.

Reply:
right, missed that 👍